addons/skin.confluence/720p/MyPVRTimers.xml

```diff
@@ -160,6 +160,14 @@
 						</control>
 						<control type="group">
 							<visible>!IsEmpty(ListItem.Date)</visible>
+							<control type="image">
+								<left>0</left>
+								<top>8</top>
+								<width>50</width>
+								<height>26</height>
+								<texture border="1">$INFO[ListItem.Icon]</texture>
```

Comment:
Do we really need a border attribute for those images? Cannot really imagine what this should improve.